lib/models/notices.js

```diff
@@ -249,7 +249,7 @@ function processCBv2Notice(service, subservice, ncr, ix) {
         var attrList = ncr.data[ix];
         for (var attr in attrList) {
             // Exclude id and type. NGSIv2
-            if (attr === 'id' || attr === 'type') {
+            if (attr !== 'id' && attr !== 'type') {
```

Comment:
Not fully sure both logical condition are equivalent... Or a bug in the logic has been solved with the change?